src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.js

```diff
@@ -442,6 +453,12 @@ export default class GeBatchGiftEntryTable extends LightningElement {
             type: this.getColumnTypeFromFieldType(element.dataType)
         };
 
+        if (column.fieldName === DONATION_AMOUNT.fieldApiName) {
+            column.typeAttributes = {
+                currencyCode: this.batchCurrencyISOCode
+            };
+        }
+
```

Comment:
Can we rename `fieldWrapper` used in this function as well as the buildColumnsFromSections() function to `fieldMapping` to make it more clear what is contained in the JSON object?